test/e2e/storage/testsuites/testdriver.go

```diff
@@ -129,6 +129,19 @@ type SnapshottableTestDriver interface {
 	GetSnapshotClass(config *PerTestConfig) *unstructured.Unstructured
 }
 
+// CustomTimeoutsTestDriver represents an interface fo a TestDriver that supports custom timeouts.
+type CustomTimeoutsTestDriver interface {
```

Comment:
just a suggestion: TestDriverWithTimeout?

Reply:
That sounds better indeed, but it'd differ from the other interfaces (they are suffixed with `TestDriver`)